tests/examples/pacer/appellate_attachment_pages/ca1_16-6001_ex1.json

```diff
@@ -0,0 +1,19 @@
+{
+  "attachments": [
+    {
+      "attachment_number": 1,
+      "description": "Main Document",
+      "pacer_doc_id": "00107463322",
+      "page_count": 7
+    },
+    {
+      "attachment_number": 2,
+      "description": "Attachment",
+      "pacer_doc_id": "00107463323",
+      "page_count": 24
+    }
+  ],
+  "case_id": "40731",
+  "main_document_id": "00107463322",
+  "pacer_seq_no": "6267366"
```

Comment:
These should align with the field names in district court, which are:

 - case_id -> pacer_case_id
 - main_document_id --> pacer_doc_id

Reply:
I knew my luck was going to run out.

Reply:
Done, field names updated.